fontbakery-check-ttf.py

```diff
@@ -1724,30 +1725,16 @@ def check_bit_entry(font, table, attr, expected, bitmask, bitname):
 
     # ----------------------------------------------------
     fb.new_check("Checking with ftxvalidator")
-    KNOWN_GOOD_OUTPUT = ''\
-                        '<?xml version="1.0" encoding="UTF-8"?>\n'\
-                        '<!DOCTYPE plist PUBLIC "-//Apple//DTD PLIST 1.0//EN"'\
-                        ' "http://www.apple.com/DTDs/PropertyList-1.0.dtd">\n'\
-                        '<plist version="1.0">\n'\
-                        '<dict>\n'\
-                        '    <key>kATSFontTestResultKey</key>\n'\
-                        '    <array>\n'\
-                        '        <string>'\
-                        'kATSFontTestSeverityInformation</string>\n'\
-                        '        <string>'\
-                        'kATSFontTestSeverityMinorError</string>\n'\
-                        '    </array>\n'\
-                        '</dict>\n'\
-                        '</plist>'
-
     try:
       ftx_cmd = ["ftxvalidator",
-                 "-r",  # Generate a full report
                  "-t", "all",  # execute all tests
                  font_file]
       ftx_output = subprocess.check_output(ftx_cmd,
                                            stderr=subprocess.STDOUT)
-      if ftx_output.strip() == KNOWN_GOOD_OUTPUT:
+
+      ftx_data = plistlib.readPlistFromString(ftx_output)
+      # we accept kATSFontTestSeverityInformation and kATSFontTestSeverityMinorError
+      if 'kATSFontTestSeverityFatalError' not in ftx_data['kATSFontTestResultKey']:
```

Comment:
I think `kATSFontTestSeverityFatalError` is the type that would prevent OSX from installing a font. The others are not considered a fail. This is also reflected in the old `KNOWN_GOOD_OUTPUT`.